src/main/kotlin/net/ccbluex/liquidbounce/utils/block/Region.kt

```diff
@@ -127,6 +128,21 @@ class Region(from: BlockPos, to: BlockPos) : ClosedRange<BlockPos>, Iterable<Blo
             this.to.z <= min.z || this.from.z >= max.z)
     }
 
+    fun getCenterPos(lowestY: Boolean = false) = Vec3d(
```

Comment:
The function description should be: `Returns the center position of the region`

But it actually is `Returns the center position of the region, but if lowestY is true, it will actually return the center of the bottom face`.

A bit silly, isn't it? haha